src/psyclone/tests/psyGen_test.py

```diff
@@ -4227,8 +4242,8 @@ def test_fparser2astprocessor_process_declarations_stmt_functions(
     reader = FortranStringReader("b(x, y) = 1")
     fparser2spec = Specification_Part(reader).content[0]
     fake_parent.symbol_table.add(Symbol('b', 'real', shape=[None, None]))
```

Comment:
Again, you may need to manually create a Statement_Function here rather than rely on fparser.

Reply:
Done